src/views/svg.js

```diff
@@ -4,11 +4,12 @@ const renderImgOrSvg = require('renderImgOrSvg');
 const debug = createDebug('app:views:svg');
 
 const svg = async (ctx, page) => {
-  const svg = await page.$eval('#container > svg', (e) =>
+  const svg = await page.$eval('#container > svg', (e) => {
     // this is a work-around by @hat215, for more details:
     // https://github.com/mermaid-js/mermaid-live-editor/issues/26#issuecomment-667678228
-    e.outerHTML.replace(/<br>/gi, '<br/>')
-  );
+    //e.style.background = 'red';
+    return e.outerHTML.replace(/<br>/gi, '<br/>');
+  });
```

Comment:
This change seems unnecessary.

Reply:
I completely is, indeed (leftover from tests).